Content.Server/_Moffstation/StationEvents/Events/VentCrittersRule.cs

```diff
@@ -83,7 +86,12 @@ protected override void Added(EntityUid uid, VentCrittersRuleComponent comp, Gam
 
         _jitter.AddJitter(location, 0.5f, 30f);
 
-        comp.SpawnAttempts ??= Math.Max(EntityQuery<VentCritterSpawnLocationComponent>().Count(), comp.SpawnAttemptsMin);
+        // Get spawn attempts
+        var playerCount = _playerManager.Sessions
+            .Count(x =>
+            GameTicker.PlayerGameStatuses.TryGetValue(x.UserId, out var status) &&
+            status == PlayerGameStatus.JoinedGame);
+        comp.SpawnAttempts = _random.Next(playerCount * comp.PlayerRatioSpawnsMin, playerCount * comp.PlayerRatioSpawnsMax);
```

Comment:
```suggestion
        // Get spawn attempts
        var playerCount = _playerManager.Sessions.Count(x =>
            GameTicker.PlayerGameStatuses.TryGetValue(x.UserId, out var status) &&
            status == PlayerGameStatus.JoinedGame);
```

`x => ...` is one expression, so breaking in the middle of it means the line needs a new indent.